tests/integration/hosts/maya/test_publish_in_maya.py

```diff
@@ -35,6 +38,36 @@ class TestPublishInMaya(MayaLocalPublishTestClass):
 
     TIMEOUT = 120  # publish timeout
 
+    def test_publish(
+        self,
+        dbcon,
+        publish_finished,
+        download_test_data
+    ):
+
```

Comment:
This could have some docstring with description why it is here.

Reply:
Done